client/src/clock-register/clock-register.directive.spec.js

```diff
@@ -26,20 +27,60 @@ describe('dojo clock register directive', function () {
     expect(element.html()).toBeDefined();
   });
 
-  describe('hit the point button', function () {
-    it('should exist', function () {
-      var hitThePointBtn = element.find('#hitThePointBtn')[0];
+  fdescribe('hitting the point', function () {
```

Comment:
I don't think we should commit with the `fdescribe`. When we have a CI running it will be crucial to run every test.